doc/reference/trd101-time.md

```diff
@@ -5,6 +5,7 @@ Kernel Time HIL
 **Working Group:** Kernel<br/>
 **Type:** Documentary<br/>
 **Status:** Final <br/>
+**Obsoleted By:** 105 <br/>
```

Comment:
```suggestion
**Status:** Obsolete <br/>
**Obsoleted By:** 105 <br/>
```

I _think_ we're supposed to do this too IIRC

Reply:
Done. I also agree it is hard to diff without pulling this change down locally and performing a diff.